src/Mirza/BusinessRegistry/Service.hs

```diff
@@ -182,7 +185,7 @@ brKeyErrorToHttpError keyError =
     UnauthorisedKeyAccessBRKE       -> httpError err403 "Not authorised to access this key."
     (PublicKeyInsertionErrorBRKE _) -> httpError err500 "Public key could not be inserted."
     (KeyNotFoundBRKE _)             -> httpError err404 "Public key with the given id not found."
-    (InvalidRevocationBRKE _ _ _)   -> httpError err500 "Key has been revoked but in an invalid way."
+    (InvalidRevocationBRKE{})       -> httpError err500 "Key has been revoked but in an invalid way."
```

Comment:
> this line needing an update
This line will only need an update when you need one of those parameters to the constructor `InvalidRevocationBRKE`, and in that case, you would have to unpack the expression anyway.
And yes, the aim was to simplify the expression and make it more maintainable, so that every update to the constructor function doesn't need someone to change the unpacking here :slightly_smiling_face: